CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+- Adds support for `ALTLIST` allowing users to represent structs as R list objects
```

Comment:
Incorrect bullet point placement. We follow [this](https://keepachangelog.com/en/1.0.0/), according to which you should put this sentence under `### Added`, like so:
```
### Added
- `ALTLIST` support, allowing users ...
```